src/wrapped_radix_engine_toolkit/high_level/manifest_building/modify_manifest.rs

```diff
@@ -149,10 +149,10 @@ impl TransactionManifest {
             }
         };
 
-        let instructions = Instructions::from_scrypto(
-            ScryptoInstructions(instructions),
+        let instructions = Instructions::try_from((
+            instructions.as_ref(),
             self.network_id(),
-        );
+        )).expect("Should not have changed depth of SBOR value, thus inserting an instruction should never fail.");
```

Comment:
we are not inserting any Instruction with too great an SBOR depth (we insert CallMethod lock fee, and TX Guarantees (WORKTOP ASSERTS))